pkg/ebpf/log.go

```diff
@@ -0,0 +1,55 @@
+package ebpf
```

Comment:
Maybe we could separate this into an `errorwriter` package?

Reply:
So I thought about it too,
but we use a very specific implementation here and I feel this struct is too small and basic for its own package.
Maybe in the future if we will develop it more than it would be exported.